yarn.lock

```diff
@@ -1417,6 +1417,15 @@
     classnames "^2.2.6"
     tslib "^1.10.0"
 
+"@fluentui/react-utils@^0.3.2-0":
```

Comment:
Looks like an issue with a version in `package.json`

Reply:
I've had this stash for a while apologies for the older things